streams/src/test/java/org/apache/kafka/streams/tests/StreamsUpgradeTest.java

```diff
@@ -269,16 +271,13 @@ public ByteBuffer encode() {
         }
 
         private ByteBuffer encodeFutureVersion() {
-            final byte[] endPointBytes = prepareUserEndPoint();
-
-            final ByteBuffer buf = ByteBuffer.allocate(getVersionThreeAndFourByteLength(endPointBytes));
+            final ByteBuffer buf = ByteBuffer.allocate(
+                4 + // used version
+                    4 // latest supported version version
+            );
 
             buf.putInt(LATEST_SUPPORTED_VERSION + 1); // used version
             buf.putInt(LATEST_SUPPORTED_VERSION + 1); // supported version
-            encodeClientUUID(buf);
-            encodeTasks(buf, prevTasks());
-            encodeTasks(buf, standbyTasks());
-            encodeUserEndPoint(buf, endPointBytes);
```

Comment:
These methods were leaked abstractions in the first place, and now they've been removed.

They should be unnecessary anyway to demonstrate that version probing works, but I'll have to run the system tests to be sure.